src/main/java/com/ucl/imaginethisserver/Conf/GsonConfig.java

```diff
@@ -20,11 +23,29 @@ public Gson gson() {
                 .create();
     }
 
+    @Bean
+    public HttpMessageConverters httpMessageConverters() {
+        GsonHttpMessageConverter gsonHttpMessageConverter = new GsonHttpMessageConverter();
+        gsonHttpMessageConverter.setGson(gson());
+        return new HttpMessageConverters(gsonHttpMessageConverter);
+    }
+
     public static class SwaggerJsonTypeAdapter implements JsonSerializer<Json> {
 
         @Override
         public JsonElement serialize(Json json, Type type, JsonSerializationContext context) {
             return JsonParser.parseString(json.value());
         }
     }
+
+//    private GsonHttpMessageConverter customGsonHttpMessageConverter() {
```

Comment:
I'd remove ignored code, or add comment saying why it might be used in the future